server/render.js

```diff
@@ -66,7 +67,7 @@ async function doRender (req, res, pathname, query, {
       Component,
       props,
       err,
-      router: new Router(pathname, query)
+      router: new Router(pathname, query, { assetPrefix })
```

Comment:
`assetPrefix` is something we use only in the client. So, we don't need to pass it here.